uv.lock

```diff
@@ -0,0 +1,856 @@
+version = 1
+revision = 1
+requires-python = ">=3.8, <4"
+resolution-markers = [
+    "python_full_version >= '3.11'",
+    "python_full_version == '3.10.*'",
+    "python_full_version == '3.9.*'",
+    "python_full_version < '3.9'",
+]
+
```

Comment:
The usage in `README.md` should also give some guidelines (even just tentative ones we can refine in the future) as to how we should manage this `uv.lock`.

Can dependabot be setup to notify us of availability of important/security upgrades?  Maybe in a future PR?

Maybe we need a more general policy on this.

Reply:
Dependabot now has support to update `uv.lock`. I didn't have a chance to try it, though.